test/functionalTest/cases/1705_csub_cache_objects/csub_cache_string_filter_update.test

```diff
@@ -287,7 +287,7 @@ Date: REGEX(.*)
 ====================================================
 POST http://localhost:9997/notify
 Content-Length: 461
-User-Agent: orion/1.0.0-next libcurl/7.19.7
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
```

Comment:
This was probably a lefover of a previous PR, which has passed unoticed until now.

NTC